### PR TITLE
Cross evm tests

### PR DESCRIPTION
**Pull request by maksym-nyrka**

### CODE REVIEW

File: functional-tests/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  },
+  "keywords": [
+    "autotests",
+    "mocha",
+    "functional"
+  ],
+  "author": "Maksym Nyrka",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/GoldenGateGGX/golden-gate/issues"
```
Comment (uchu) [inline functional-tests/package.json:22]: As we renamed our github, probably it makes sense to use ggxchain instead og GoldenGateGGX

Just a minor note

**Review by uchu — APPROVED** [review]
Looks good! Approved with minor notes regarding the repository url in the package. However, it shouldn't affect anything.

File: functional-tests/src/common/СommonEvm.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            const input = {
+                language: "Solidity",
+                sources: {main: {content: sourceCode}},
+                settings: {outputSelection: {"*": {"*": ["abi", "evm.bytecode"]}}},
+            };
+
+            // Parse the compiler output to retrieve the ABI and Bytecode
+            const output = solc.compile(JSON.stringify(input));
+            console.log('output')
+            console.log(output)
```
Comment (obsessed-cake) [inline functional-tests/src/common/СommonEvm.js:49]:
```suggestion
            console.log(`output\n${output}`)
```

File: functional-tests/package.json (hunk 1 above)
Comment (maksym-nyrka) [reply functional-tests/package.json:22]: Thanks, will update it.

**Review by obsessed-cake — APPROVED** [review]

**Pull request merged**